module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/ComponentTest.php

```diff
@@ -50,7 +51,8 @@ class ComponentTest extends \PHPUnit\Framework\TestCase
      */
     protected function getHelper()
     {
-        $helper = new Component();
+        $themeInfoMock = $this->getMockBuilder(ThemeInfo::class)->disableOriginalConstructor()->getMock();
```

Comment:
You can simplify to:

```suggestion
        $themeInfoMock = $this->createMock(ThemeInfo::class);
```